### DISCUSSION TITLE
Adding arm64 to 4k-video-downloader cask

### DISCUSSION THREAD

**Discussion by srobadelic** [Homebrew/discussions#5796] [Everyday usage]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

4k-video-downloader is only installing intel version, but there is an ARM64 version (for Apple Silicon). I wasn't certain where to put this because it's not an error any there are no issues other than the Cask could be updated to include the detect and download for arm64.

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Updating-Software-in-Homebrew should help you adjust that

**Reply by gromgit** [reply]
https://docs.brew.sh/Cask-Cookbook#handling-different-system-configurations should also be useful.

**Comment by daeho-ro** [comment]
Are you talking about `4k-video-downloader-plus`? I haven't found arm version for `4k-video-downloader`.

**Comment by srobadelic** [comment]
I just noticed this difference. Thank you! And there is a cask for that which works great. I didn’t know there was a plus. Thank you all!

SS
